content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -874,6 +878,12 @@ To differentiate between these possibilities, we've introduced three _null enums
 - `DbNull`: Selects the `NULL` value in the database.
 - `AnyNull`: Selects both `null` JSON values and `NULL` database values.
 
+<Admonition type="info">
+
+From v4.0.0, Prisma returns `JsonNull`, `DbNull`, and `AnyNull` as [JavaScript symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol). Before v4.0.0, Prisma returned them as strings.
```

Comment:
We don't actually return them anywhere, they are only used as inputs. Both kinds of null values are returned as JavaScript `null`s in the results.

Maybe something like:

```suggestion
From v4.0.0, `JsonNull`, `DbNull`, and `AnyNull` are [JavaScript symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol). Before v4.0.0, they were strings.
```